src/main/java/pl/piekoszek/gorskimatches/equation/CharacterChanger.java

```diff
@@ -0,0 +1,12 @@
+package pl.piekoszek.gorskimatches.equation;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class CharacterChanger {
+    public String changeCharactersInString(String string, int position, char character) {
+        char[] inputStringCharArr = string.toCharArray();
```

Comment:
**minor**
I'd just call it `characters`

Reply:
ok